src/frontend/src/lib/components/tokens/NoTokensPlaceholder.svelte

```diff
@@ -0,0 +1,24 @@
+<script lang="ts">
+	import icpDark from '$eth/assets/icp_dark.svg';
+	import bitcoin from '$icp/assets/bitcoin.svg';
+	import eth from '$icp-eth/assets/eth.svg';
+	import Logo from '$lib/components/ui/Logo.svelte';
+	import { i18n } from '$lib/stores/i18n.store';
+</script>
+
+<div class="py-12">
```

Comment:
Forgot this in my last review, sorry: a transition here maybe? how does it behaves on transition?

Reply:
I believe the Logo component already takes care of this: `<div
	class="flex items-center justify-center overflow-hidden rounded-full ring-white"
	class:bg-dust={color === 'dust' && !loaded}
	class:bg-off-white={color === 'off-white' && !loaded}
	class:bg-white={color === 'white' && !loaded}
	class:opacity-10={!loaded}
	class:ring-2={ring}
	style={`width: ${sizePx}; height: ${sizePx}; transition: opacity 0.15s ease-in;`}
>`

But.. good point about the loading time. This leads to a undesired behaviour:
https://github.com/user-attachments/assets/f76c88a1-54ef-4c3b-bbf2-58ad0d5ffed4

I added very small z-indexes to ensure the stacking. Here's a quick video, also ensuring modal functionality still works.
https://github.com/user-attachments/assets/7fda8d00-f65d-4fb6-9f9f-8fafe1539973